test/integration/operator/integration_suite_test.go

```diff
@@ -129,6 +129,11 @@ var _ = ginkgo.BeforeSuite(func() {
 	_, err = operatorClient.OperatorV1().ClusterManagers().Create(context.Background(), &operatorapiv1.ClusterManager{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: clusterManagerName,
+			Labels: map[string]string{
+				"app":      "clustermanager",
```

Comment:
same here, consider a test that update the clustermanager, and check that the label is updated on the resource instead of setting it globally.